tests/kafkatest/services/console_consumer.py

```diff
@@ -61,7 +61,8 @@ def __init__(self, context, num_nodes, kafka, topic, group_id="test-consumer-gro
                  message_validator=None, from_beginning=True, consumer_timeout_ms=None, version=DEV_BRANCH,
                  client_id="console-consumer", print_key=False, jmx_object_names=None, jmx_attributes=None,
                  enable_systest_events=False, stop_timeout_sec=15, print_timestamp=False,
-                 isolation_level="read_uncommitted", jaas_override_variables=None):
+                 isolation_level="read_uncommitted", jaas_override_variables=None,
+                 kafka_opts_override="", consumer_properties={}, client_prop_file_override=""):
```

Comment:
Do we need to both  consumer_properties and client_prop_file_override? maybe we can just use client_prop_file_override